functional-tests/base-test.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
Not sure if these are needed, they weren't included in saucelabs example but I don't know how the travis env would be able to run selenium otherwise.